src/lib/Containers/Artist.tsx

```diff
@@ -93,11 +96,11 @@ export class Artist extends Component<Props & ViewProperties, State> {
   renderSelectedTab = () => {
     switch (this.selectedTabTitle()) {
       case TABS.ABOUT:
-        return <About artist={this.props.artist} />
+        return <About artist={this.props.artist as any} />
       case TABS.WORKS:
-        return <Artworks artist={this.props.artist} />
+        return <Artworks artist={this.props.artist as any} />
       case TABS.SHOWS:
-        return <Shows artist={this.props.artist} />
+        return <Shows artist={this.props.artist as any} />
```

Comment:
I assume these additional `as any` are due to the relay data masking, I think @kastermester covers [that here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23460#pullrequestreview-94403153)

Reply:
Aye 👍

Reply:
Btw, the only reason that in some cases we didn’t need to specify `as any` previously was that those components were overriding the constructor without re-typing the props, which meant that the component basically accepted anything by default.

Reply:
The future plans tie in with the way the artifacts are generated - it means that you cannot pass data to a component, without having actually specified the fragment as part of your fragment text (ie. you need to spread the child component's fragment into your own at the appropriate place to be able to render the child component with the given data).

As mentioned, we're waiting for conditional types coming in TS 2.8 (and conditional infered types as well to be able to type up the `componentRef` prop).